### PR TITLE
Add JavaDoc to BREAD-Common

### PR DESCRIPTION
**Pull request by Sunbread**
Improve readability.

### CODE REVIEW

File: common/src/main/java/top/sunbread/bread/common/BREADAnalyser.java [hunk 1]
```diff
@@ -70,32 +80,59 @@ public BREADAnalyser(Map<UUID, Set<Point>> points, boolean fast,
         this.future = mapFuture.thenApply(Optional::of).acceptEither(timeout(TIMEOUT_MINUTES, TimeUnit.MINUTES),
                 result -> {
                     this.threadPool.shutdownNow();
-            asyncCallback.accept(result);
-        });
+                    asyncCallback.accept(result);
```
Comment (Sunbread) [inline common/src/main/java/top/sunbread/bread/common/BREADAnalyser.java]: Unexpected formatting :joy:

File: common/src/main/java/top/sunbread/bread/common/BREADAnalyser.java [hunk 2]
```diff
@@ -70,32 +80,59 @@ public BREADAnalyser(Map<UUID, Set<Point>> points, boolean fast,
...
+                });
```
Comment (Sunbread) [inline common/src/main/java/top/sunbread/bread/common/BREADAnalyser.java:84]: Unexpected formatting :joy:

**Review by Sunbread — COMMENTED** [review]
All problems have been resolved.

**Review by DWVoid — APPROVED** [review]
Comprehensible wording, but might be a little vague for some functions. Please consider incremental wording improvements in future versions.

**Pull request merged**